src/components/FooterV2.js

```diff
@@ -65,6 +68,12 @@ const useStyles = makeStyles((theme) => ({
     color: '#fff',
     width: theme.spacing(30),
   },
+  subColumnTwo: {
+    display: 'flex',
+    flexDirection: 'column',
+    color: '#fff',
+    width: theme.spacing(19),
+  },
```

Comment:
At a medium-width, it's an unpleasant layout:
<img width="1434" alt="Screen Shot 2021-11-16 at 5 32 53 PM" src="https://user-images.githubusercontent.com/5924325/142082894-62e59da4-f39c-4974-9db8-9bf9ebadc9a2.png">

Reply:
the hard part here is that I'm leaving space for an image on the right side so if I leave the links on the top, it gets really cramped

Reply:
Maybe the image is always on the right but flexes smaller, hen if any sections reflow, they all do? So this case would look like:

| [logo] |  |
| ------ | ------ |
| [social] | [image] |
| [links] |  |